src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/CodeStyleHelpers.cs

```diff
@@ -13,6 +13,13 @@ namespace Microsoft.CodeAnalysis.CodeStyle
 {
     internal static class CodeStyleHelpers
     {
+        /// <summary>
+        /// Delimiters which the compiler previously treated as embedded comment delimiters for parsing EditorConfig
+        /// files. For code style options maintained by Roslyn, we continue to detect and remove these "comments" for
+        /// backwards compatibility.
+        /// </summary>
+        private static readonly char[] s_commentDelimiters = ['#', ';'];
```

Comment:
Did we want to share this code/list with the compiler code doing the same thing? The file path here makes me think we've already got shared files between both layers to use here.

Reply:
➡️ The sharing here is between the workspaces and code style layers. The compiler maintains its own back-compat separately.